staticconf/config.py

```diff
@@ -439,6 +439,31 @@ def has_changed(self):
         return last_mtime < self.last_max_mtime
 
 
+def create_LoggingMTimeComparator(err_logger):
```

Comment:
This doesn't necessarily need to be part of this repo, but if it stays the pythonic name for this should be something like `build_logging_mtime_comparator()` .  (`build_x` is a commonly used function name in this library)